cmd/stack.go

```diff
@@ -185,6 +224,11 @@ func init() {
 	stackUpdateCmd.Flags().BoolVarP(&forceStack, "force", "f", false, "force override previous deployment")
 	utils.CheckErr(stack.AddOptions(stackUpdateCmd, false))
 
+	stackCmd.AddCommand(command.AddDependencyCheck(stackPreviewCommand, command.Pulumi, command.Docker))
+	stackPreviewCommand.Flags().StringVarP(&envFile, "env-file", "e", "", "--env-file config/.my-env")
+	stackPreviewCommand.Flags().BoolVarP(&forceStack, "force", "f", false, "force override previous deployment")
```

Comment:
Does preview need a force?

Reply:
If its in a conflicting state then it needs a force